app/dlr_limit_consumer.py

```diff
@@ -0,0 +1,91 @@
+from kafka import KafkaConsumer
+from json import loads
+import time
+import csv
+import sys
+import os
+
+show_debug = True
+show_data = True
+
+# Function to call the kafka consumer and taking the last element
+def consumer_kafka_to_csv():
+    # buffer = []
+    for message in consumer:
+        print('I were here')
```

Comment:
```suggestion
        print('Got message from kafka:')
        print(message.value)
```
If you want to print each time you read a message, it might be smart to include something that changes, otherwise it can be quite hard to see when a new line is added at the end.